api/exception/handlers.py

```diff
@@ -46,8 +53,8 @@ def global_exception_handler(err):
     return __handler_response(code, 'An error occurred while trying to complete your request. Please try again later. If the problem persists, please contact support for further assistance.')
 
 
-def __handler_response(code, description='Something went wrong'):
-    response = {'code': code, 'message': description}
+def __handler_response(code, description='Something went wrong', **kwargs):
```

Comment:
question: Are we sure we want to return the input validation errors as they are? What could this information be used for?

Reply:
Yeah but validation errors will not be handled by the frontend, will they? Either way, I agree there is not sensitivi info